hazelcast/src/main/java/com/hazelcast/internal/management/ThreadDumpGenerator.java

```diff
@@ -51,7 +51,7 @@ private static String dump(ThreadInfo[] infos, StringBuilder s) {
         header(s);
         appendThreadInfos(infos, s);
         if (LOGGER.isFinestEnabled()) {
-            LOGGER.finest("\n" + s.toString());
+            LOGGER.finest("\n" + s);
```

Comment:
I would require some other changes and it isn't the intend of this particular PR. I just removed code which is going to be added by the javac anyway.